### DISCUSSION TITLE
it it fixed?

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#111] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
do u have a guess of how long this gon take

**Comment by The-Man-Is-Me** [comment]
https://github.com/Holding-THE-CREATOR/Pingot 
said they needed at least a week, 6 days ago, really depends on the want and support for the exploit.

**Comment by 3kh0** [comment, marked as answer]
A workaround has been started, please be patient.
